distribution/dashboard/ui/src/services/index.ts

```diff
@@ -183,16 +183,18 @@ export async function getClusterDetailReq({
   ns,
   name,
   useFor,
+  tenantTopoData
 }: {
   ns: string;
   name: string;
   useFor?: string;
+  tenantTopoData?:API.ReplicaDetailType[]
 }) {
   const r = await request(`/api/v1/obclusters/namespace/${ns}/name/${name}`, {
     method: 'GET',
   });
   if (r.successful) {
-    if (useFor === 'topo') return formatTopoData(r.data);
+    if (useFor === 'topo') return formatTopoData(r.data,tenantTopoData);
```

Comment:
`formatTopoData` seems to be renamed to `formatTenantData` and is more consistent with formatClusterData later.

Reply:
I got you,but the `formatTopoData` is utilized not just by tenants but also by clusters.